control-plane/roles/logging/templates/loki-stack-values.yaml

```diff
@@ -0,0 +1,8 @@
+---
+grafana:
+  enabled: false
+
+# promtail:
+#   initContainer:
+#     enabled: true
+#     fsInotifyMaxUserInstances: 10
```

Comment:
Can be removed.

Reply:
https://github.com/metal-stack/metal-roles/commit/a487604d0dca7eb6be1b30fb68df2dc0cd8286b6